examples/stable_diffusion_v2/inference/config/controlnet_canny.yaml

```diff
@@ -0,0 +1,18 @@
+image_path: "path/to/test_imgs/dog2.png"
+prompt: "cute dog"  # The prompt to render
+controlnet_mode: "canny" # candidates: [segmentation, canny]
+a_prompt: "best quality, extremely detailed"
+negative_prompt: "longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality"  # The negative prompt not to render
+H: 512  # Image height, in pixel space
+W: 512  # Image width, in pixel space
+image_resolution: 512
+low_threshold: 100
+high_threshold: 200
+strength: 1
+guess_mode: False
+condition_ckpt_path: "models/deeplabv3plus_s16_ascend_v190_voc2012_research_cv_s16acc79.06_s16multiscale79.96_s16multiscaleflip80.12.ckpt"
```

Comment:
can we remote it? canny control doesn't need this ckpt

Reply:
fixed